recipes/gcc/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "10.2.0":
+    sha256: 27e879dccc639cd7b0cc08ed575c1669492579529b53c9ff27b0b96265fa867d
+    url: https://ftp.gnu.org/gnu/gcc/gcc-10.2.0/gcc-10.2.0.tar.gz
```

Comment:
If this turns out to be unreliable we might want to add mirrors https://gcc.gnu.org/mirrors.html

but for now lets see if this is reliable enough

Reply:
Official GCC docker image has a good mirror range listed: https://github.com/docker-library/gcc/blob/36c1ff6d7b44428b35fa8b61787c76b225d8184a/10/Dockerfile